test/specs/smoke-test.js

```diff
@@ -0,0 +1,83 @@
+import loginPage from '../pageobjects/login.page.js';
+import securePage from '../pageobjects/secure.page.js';
+
+import LoginPage from '../pageobjects/login.page.js';
+
+
+describe('Smoke test for Shop Tools App', () => {
```

Comment:
Please add test suite and test case IDs

i.e. '001: Smoke test for Shop Tools App'